### DISCUSSION TITLE
[Enhancement] [date-picker] Allow month/year selection when type="daterange"

### DISCUSSION THREAD

**Discussion by hahatulk** [element-plus/element-plus#10637] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

Please add prop to enable "year/month" selection like it is now in type="date", when you click title of panel e.g. "2022", "November"

### Additional comments

_(empty)_

**Comment by cccMing** [comment]
I have the same question. Inconvenient to use，我看其他组件库类似功能都支持的

**Comment by wjp980108** [comment]
进行中：[PR](https://github.com/element-plus/element-plus/pull/19700)

**Comment by wjp980108** [comment]
Released, please update to 2.9.11
